packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -113,6 +113,7 @@ export const LoadedEditor: React.FC<LoadedEditorProps> = ({
       jsx,
       parseTSConfig(tsconfig).compilerOptions,
     );
+    // @ts-expect-error Monaco typescript.CompilerOptions is incompatible with typescript 5.0 types
```

Comment:
We can remove this when Monaco updates `CompilerOptions` type

Reply:
Out of curiousity, what's incompatible here? I'm trying to get monaco up to date among other things, so I'm curious.

Monaco has to redeclare everything for their public API, so, it's likely that this happens a lot and casing this away forever might be better (unless the problem is trivial).

Reply:
Without `@ts-expect-error`, TS output the error `Type 'ModuleResolutionKind.Classic' is not assignable to type 'ModuleResolutionKind | undefined'.`